app/presenters/content_item_presenter.rb

```diff
@@ -86,5 +86,31 @@ def details
       show_summaries: false,
       facets: [],
     }
+
+    details.merge(nation_applicability)
+  end
+
+  def nation_applicability
+    if policy.applicable_nations.length < policy.possible_nations.length
+      {
+        nation_applicability: {
+          applies_to: policy.applicable_nations,
+          alternative_policies: alternative_policies,
+        }
+      }
+    else
+      {}
+    end
+  end
+
+
+  def alternative_policies
+    inapplicable_nations = policy.possible_nations - policy.applicable_nations || []
```

Comment:
Reckon it's worth pulling this out to a `inapplicable_nations` method. That can be used in the `if` in the method above as well - `if inapplicable_nations.any?`.